### ISSUE TITLE
Git blame format list the filepath too long.

### ISSUE THREAD

**Issue by zhchaoo** [kemayo/sublime-text-git#516]
for me, the most information of bit blame is sha commiter and date, but now the git blame list sha filepath first, and the filepath always is very long, I couldn't get my concerned info in my first eye.

**Comment by Hitabis** [comment]
I agree. Most of all, the file path is redundant when running git blame on a single file.
It should be shown at the top and not in every single line

**Comment by scottschup** [comment]
+1
